spec/integration/manifest.spec.yml

```
@@ -217,8 +217,11 @@ spec:
             description: Link title
             type: string
           url:
-            description: Link url
+            description: Link url. Format is `http://...` or `https://...` for external links,  `kbn:app/...` for links internal to Kibana.
             type: string
+            oneOf:
+              - type: string
+                pattern: '^(http(s)?://|kbn:/)'
```

`oneOf` is not needed now, as there is a single option.
```suggestion
            type: string
            pattern: '^(http(s)?://|kbn:/)'
```